.github/workflows/update-references.yml

```diff
@@ -0,0 +1,37 @@
+name: update references
+on: create
+
+jobs:
+  update-reference:
+    if: ${{ contains(github.ref, 'refs/heads/release') }}
```

Comment:
Ok I updated this following this pattern: https://github.community/t/trigger-job-on-branch-created/16878/5 so I think it should happen on branch creation.